.capistrano/metrics.example

```diff
@@ -0,0 +1 @@
+none
```

Comment:
We should probably not check this file in so users can decide whether to send metrics.